File: Elyes/src/CiteBundle/Entity/Stand.php
<?php

namespace CiteBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Stand
 *
 * @ORM\Table(name="stand")
 * @ORM\Entity
 */
class Stand
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id_stand", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idStand;

    /**
     * @var string
     *
     * @ORM\Column(name="titre_stand", type="string", length=255, nullable=false)
     */
    private $titreStand;

    /**
     * @var string
     *
     * @ORM\Column(name="proprietaire_stand", type="string", length=255, nullable=false)
     */
    private $proprietaireStand;

    /**
     * @var string
     *
     * @ORM\Column(name="type_marchandise", type="string", length=255, nullable=false)
     */
    private $typeMarchandise;


    /**
     * @var integer
     *
     * @ORM\Column(name="taille", type="integer", nullable=true)
     */
    private $taille;

//------------------------------------------------------------ Getters and Setters ---------------------------------------------

    /**
     * @return int
     */
    public function getIdStand()
    {
        return $this->idStand;
    }

    /**
     * @param int $idStand
     */
    public function setIdStand($idStand)
    {
        $this->idStand = $idStand;
    }

    /**
     * @return string
     */
    public function getTitreStand()
    {
        return $this->titreStand;
    }

    /**
     * @param string $titreStand
     */
    public function setTitreStand($titreStand)
    {
        $this->titreStand = $titreStand;
    }

    /**
     * @return string
     */
    public function getProprietaireStand()
    {
        return $this->proprietaireStand;
    }

    /**
     * @param string $proprietaireStand
     */
    public function setProprietaireStand($proprietaireStand)
    {
        $this->proprietaireStand = $proprietaireStand;
    }

    /**
     * @return string
     */
    public function getTypeMarchandise()
    {
        return $this->typeMarchandise;
    }

    /**
     * @param string $typeMarchandise
     */
    public function setTypeMarchandise($typeMarchandise)
    {
        $this->typeMarchandise = $typeMarchandise;
    }


    /**
     * @return string
     */
    public function getTaille()
    {
        return $this->taille;
    }

    /**
     * @param string $actif
     */
    public function setTaille($taille)
    {
        $this->taille = $taille;
    }






}

File: CodenameOneConsommationWebService/src/com/mycompany/Entite/Stand.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.Entite;

import java.util.Date;

/**
 *
 * @author Elyes
 */
public class Stand {

    int idStand;
    String proprietaireStand;
    String typeMarchandise;
    String titreStand;
    int taille;
    fos_user user;

// ------------------------------------------------ Constractors ---------------------------------------------    
    public Stand(int idStand, String proprietaireStand, String typeMarchandise, String titreStand, int taille, fos_user user) {
        this.idStand = idStand;
        this.proprietaireStand = proprietaireStand;
        this.typeMarchandise = typeMarchandise;
        this.titreStand = titreStand;
        this.taille = taille;
        this.user = user;
    }

    public Stand(int idStand, String proprietaireStand, String typeMarchandise, String titreStand, int taille) {
        this.idStand = idStand;
        this.proprietaireStand = proprietaireStand;
        this.typeMarchandise = typeMarchandise;
        this.titreStand = titreStand;
        this.taille = taille;
    }
    
    

    public Stand() {
    }

// ------------------------------------------------ Getters and Setters -------------------------------------
    public int getIdStand() {
        return idStand;
    }

    public void setIdStand(int idStand) {
        this.idStand = idStand;
    }

    public String getProprietaireStand() {
        return proprietaireStand;
    }

    public void setProprietaireStand(String proprietaireStand) {
        this.proprietaireStand = proprietaireStand;
    }

    public String getTypeMarchandise() {
        return typeMarchandise;
    }

    public void setTypeMarchandise(String typeMarchandise) {
        this.typeMarchandise = typeMarchandise;
    }

    public String getTitreStand() {
        return titreStand;
    }

    public void setTitreStand(String titreStand) {
        this.titreStand = titreStand;
    }

    public int getTaille() {
        return taille;
    }

    public void setTaille(int taille) {
        this.taille = taille;
    }

    public fos_user getUser() {
        return user;
    }

    public void setUser(fos_user user) {
        this.user = user;
    }

// ------------------------------------------------ ToString -------------------------------------    
    @Override
    public String toString() {
        return "Stand{" + "idStand=" + idStand + ", proprietaireStand=" + proprietaireStand + ", typeMarchandise=" + typeMarchandise + ", titreStand=" + titreStand + ", taille=" + taille + ", user=" + user + '}';
    }

}
